server/public/model/channel_member.go

```diff
@@ -6,22 +6,24 @@ package model
 import (
 	"net/http"
 	"strings"
+	"unicode/utf8"
 )
 
 const (
-	ChannelNotifyDefault            = "default"
-	ChannelNotifyAll                = "all"
-	ChannelNotifyMention            = "mention"
-	ChannelNotifyNone               = "none"
-	ChannelMarkUnreadAll            = "all"
-	ChannelMarkUnreadMention        = "mention"
-	IgnoreChannelMentionsDefault    = "default"
-	IgnoreChannelMentionsOff        = "off"
-	IgnoreChannelMentionsOn         = "on"
-	IgnoreChannelMentionsNotifyProp = "ignore_channel_mentions"
-	ChannelAutoFollowThreadsOff     = "off"
-	ChannelAutoFollowThreadsOn      = "on"
-	ChannelAutoFollowThreads        = "channel_auto_follow_threads"
+	ChannelNotifyDefault             = "default"
+	ChannelNotifyAll                 = "all"
+	ChannelNotifyMention             = "mention"
+	ChannelNotifyNone                = "none"
+	ChannelMarkUnreadAll             = "all"
+	ChannelMarkUnreadMention         = "mention"
+	IgnoreChannelMentionsDefault     = "default"
+	IgnoreChannelMentionsOff         = "off"
+	IgnoreChannelMentionsOn          = "on"
+	IgnoreChannelMentionsNotifyProp  = "ignore_channel_mentions"
+	ChannelAutoFollowThreadsOff      = "off"
+	ChannelAutoFollowThreadsOn       = "on"
+	ChannelAutoFollowThreads         = "channel_auto_follow_threads"
+	ChannelMemberNotifyPropsMaxRunes = 800000
```

Comment:
How did we come up with this number?

Reply:
It is the same one we're using [in the post props](https://github.com/mattermost/mattermost/blob/master/server/public/model/post.go#L60), a maximum of 3.2MB